llvm/lib/Transforms/Vectorize/VPlan.cpp

```diff
@@ -474,6 +474,14 @@ void VPIRBasicBlock::execute(VPTransformState *State) {
     // backedges. A backward successor is set when the branch is created.
     const auto &PredVPSuccessors = PredVPBB->getHierarchicalSuccessors();
     unsigned idx = PredVPSuccessors.front() == this ? 0 : 1;
+    if (TermBr->getSuccessor(idx) &&
```

Comment:
I think this code was for a different version of the patch using BranchOnMultiCond - can be deleted I think?